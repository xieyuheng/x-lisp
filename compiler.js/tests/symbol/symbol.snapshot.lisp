'abc
(symbol-length 'abc)
(symbol-to-string 'abc)
