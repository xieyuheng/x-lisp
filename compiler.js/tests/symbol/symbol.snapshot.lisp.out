'abc
3
"abc"
